vunit/ui.py

```diff
@@ -1888,6 +1896,10 @@ def __init__(self, source_file, project, ui):
         self._project = project
         self._ui = ui
 
+    @property
+    def project_source_file(self):
```

Comment:
This exposes a private class on the public API. Use private access instead with a pylint disable comment.